BIPs/2023-W21/BIP-308 mainnet.json

```diff
@@ -0,0 +1,68 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1685029585377,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.14.1",
+    "createdFromSafeAddress": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f",
+    "createdFromOwnerAddress": "",
+    "checksum": "0xa91f3b6ac5501d18d8c97d43f5e5709fa958e1493a097e17ae951b8245f2139a"
+  },
+  "transactions": [
+    {
+      "to": "0xA331D84eC860Bf466b4CdCcFb4aC09a1B43F3aE6",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          { "internalType": "bytes32", "name": "role", "type": "bytes32" },
+          { "internalType": "address", "name": "account", "type": "address" }
+        ],
+        "name": "grantRole",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "role": "0xaecef2a08acfa6437c6cad5d0aad2bd0172fec6050bd95d13aa5450c25aaa391",
+        "account": "0x10A19e7eE7d7F8a52822f6817de8ea18204F2e4f"
+      }
+    },
+    {
+      "to": "0x96484f2aBF5e58b15176dbF1A799627B53F13B6d",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "contract IOmniVotingEscrow",
+            "name": "omniVotingEscrow",
+            "type": "address"
+          }
+        ],
+        "name": "setOmniVotingEscrow",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "omniVotingEscrow": "0xE241C6e48CA045C7f631600a0f1403b2bFea05ad"
```

Comment:
Looking for clarifiaction here:

0xE24 doesn't show up in the deployments addresses.

```
    '20230504-vebal-remapper/OmniVotingEscrowAdaptor': "0x96484f2aBF5e58b15176dbF1A799627B53F13B6d"
```
however can be found.  Is this address correct?  Where can the omniVotingEscrow address be verified.

Reply:
@jubeira. Any ideas here?

Reply:
https://github.com/BalancerMaxis/bal_addresses/pull/35